e2e/serial/swap/1_swapFlow1.test.ts

```diff
@@ -108,6 +108,11 @@ it('should be able to connect to hardhat', async () => {
   const button = await findElementByText(driver, 'Disconnect from Hardhat');
   expect(button).toBeTruthy();
   await findElementByTestIdAndClick({ id: 'navbar-button-with-back', driver });
+
+  // sometimes hardhat balances take time to display.
+  // If we go straight to swap and they aren't updated,
+  // the swap gets auto-populated with the wrong token.
+  await delay(10_000);
```

Comment:
was seeing failures maybe 10% of the time bc of this. Feels better to have it than not.